packages/arb-bridge-eth/contracts/rollup/IRollupCore.sol

```diff
@@ -71,7 +73,16 @@ interface IRollupCore {
         uint256 finalBalance
     );
 
-    function getNode(uint256 nodeNum) external view returns (INode);
+    /**
+     * @notice Get the Node properties for the given index. 
+     * We can't expose the full Node publicly as it contains a mapping 
+     */
+    function getNodeProps(uint256 nodeNum) external view returns (NodeProps memory);
```

Comment:
Any reason to change getNode to getNodeProps?
This doesn't need to be a breaking change. Could instead flatten the 2 node structs into 1, or call the outer node "NodeWithStaker" and inner "Node" to keep previous interface

Reply:
Had NodeWithStaker originally, happy to use that. I think I'm in favour of going with your idea of completely separating the stakers into their own mapping

Reply:
We now have a public getNode and an internal getNodeStorage.